### ISSUE TITLE
Core: KeyboardEvent.which/MouseEvent.which are deprecated

### ISSUE THREAD

**Issue by melloware** [primefaces/primefaces#9650]
`KeyboardEvent.which` is deprecated and should be replaced with `KeyboardEvent.key` everywhere.

Here is a great Keyboard event viewer: https://w3c.github.io/uievents/tools/key-event-viewer.html

Also its universally supported now: https://caniuse.com/?search=event.key

Also changed `MouseEvent.which` to `MouseEvent.button`
